futures-util/src/try_future/and_then.rs

```diff
@@ -3,10 +3,7 @@ use core::mem::PinMut;
 use futures_core::future::{Future, TryFuture};
 use futures_core::task::{self, Poll};
 
-/// Future for the `and_then` combinator, chaining a computation onto the end of
-/// another future which completes successfully.
-///
-/// This is created by the `Future::and_then` method.
+/// Future for the [`and_then`](super::TryFutureExt::and_then) combinator.
```

Comment:
I replaced all these with just a link to the detailed documentation of the combinator method. This avoids duplication and is better for the user because the method's documentation far better.